piecrust/CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Derive `Debug` for `Session` and `VM`
```

Comment:
Shouldn't this be for `ModuleStore` and `VM` ?

Reply:
I don't know what happened but some of my changes weren't in this commit. Weird. I only put structs that are a part of the API in the changelog.